modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalAddFilter.tsx

```diff
@@ -413,6 +413,17 @@ export function ModalAddFilter({
 			return true;
 		}
 	};
+	const isRelationshipFieldFilterBy = () => {
```

Comment:
@danvb112 @p-albuquerque I don't think this needs to be a function. You can replace it with the following:

```
const aggregationRelationshipOrDateFieldType =
        selectedFilterBy?.businessType === 'Date' ||
	(aggregationFilter &&
	       selectedFilterBy?.businessType === 'Relationship');
```

And instead of `selectedFilterBy && isRelationshipFieldFilterBy() &&(` you call 
`{selectedFilterBy && !aggregationRelationshipOrDateFieldType && (`